src/step07.rb

```diff
@@ -153,8 +154,10 @@ def self.texture_to_hash(texture)
       pixel_height: texture.image_height,
       model_width: texture.width,
       model_height: texture.height,
-      model_width_formatted: texture.width.to_l.to_s,
-      model_height_formatted: texture.height.to_l.to_s,
+      # Replace spaces with non-breaking spaces in lengths to avoid
+      # "~" sign or unit from being on a different line tha the number.
+      model_width_formatted: texture.width.to_l.to_s.gsub(" ", "\u00A0"),
```

Comment:
Instead of manipulating the strings - I'd think we can use CSS to control this. (This feels like changing the source text to all capitals when you can instead apply a `text-transform`)

https://developer.mozilla.org/en-US/docs/Web/CSS/white-space
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Might need to wrap a span around the text where we don't want the lines to break.